ledfx/sendspin/stream.py

```
@@ -152,7 +181,16 @@ def _audio_chunk_handler(
                 )
 
         except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)